app/Http/Controllers/TrainingController.php

```diff
@@ -67,8 +68,9 @@ public function create()
             $qualifications = Qualification::where('client_id', '=', Auth::user()->currentclient_id)->orderBy('name')->get();
             $positions = new \Illuminate\Database\Eloquent\Collection(); //initial no positions
             $training_users = new \Illuminate\Database\Eloquent\Collection(); //initial no training_users
+            $credit = new \Illuminate\Database\Eloquent\Collection(); //initial no credit
```

Comment:
Perfect on create!
We have also to deliver this on edit so the view can show the actual state in the database